src/deadline/client/config/config_file.py

```diff
@@ -33,7 +33,8 @@
 # The default AWS Deadline Cloud endpoint URL
 # Environment variable that, if set, overrides the value of DEFAULT_DEADLINE_ENDPOINT_URL
 DEFAULT_DEADLINE_ENDPOINT_URL = os.getenv(
```

Comment:
Not related to your change, but reading an AWS SDK env var like this smells to me, why would we do this? This doesn't look like it is used anyway

Reply:
Agreed, this would be a great refactor PR to apply.